frontend/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM node:22-alpine
```

Comment:
Run it over  node:20-alpine please

Reply:
@yesahem It doesn't matter it is independent of the OS. Whether you use alpine image or the debian image.